resources/layout/main.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+use App\Asset\AppAsset;
+use Yiisoft\Html\Html;
+
+$assetManager->register([
```

Comment:
Should be declared via phpdoc. Same should be done for layoutParameters, content and csrf.

Reply:
done.